### PR TITLE
7887: sort todays orders

### PR DESCRIPTION
**Pull request by kkoskelin**
Enable the ability to sort by filing date OR number of pages (ascending, descending)

NOTE:  this PR will require the re-indexing of docket entries before all code will function correctly, to sort by number of pages.

### CODE REVIEW

File: shared/src/business/useCases/filePetitionInteractor.js [hunk 1]
```diff
@@ -81,8 +81,7 @@ exports.filePetitionInteractor = async ({
 
   const caseDetail = await applicationContext
     .getUseCases()
-    .createCaseInteractor({
-      applicationContext,
+    .createCaseInteractor(applicationContext, {
```
Comment (kkoskelin) [inline shared/src/business/useCases/filePetitionInteractor.js]: This change was an overlooked refactor bug that I noticed and had to address, unrelated to 7887.

Comment (kkoskelin) [reply shared/src/business/useCases/filePetitionInteractor.js]: This change was erroneous -- removed.

File: web-client/src/presenter/computeds/public/todaysOrdersHelper.js [hunk 2]
```diff
@@ -1,6 +1,24 @@
 import { state } from 'cerebral';
 
 export const todaysOrdersHelper = (get, applicationContext) => {
+  const { TODAYS_ORDER_SORT } = applicationContext.getConstants();
+
+  const sortOptions = [
```
Comment (ericsorenson) [inline web-client/src/presenter/computeds/public/todaysOrdersHelper.js:9]: Move to a constants file?

**Review by ericsorenson — APPROVED** [review]

**Pull request merged**